scripts/test-all.sh

```
@@ -0,0 +1,88 @@
+        echo "⚠ Go not found, skipping Go tests"
+    fi
+else
+    echo "⚠ Go implementation directory not found, skipping Go tests"
+fi
+echo ""
+
+# Run TypeScript tests
+echo "--- Running TypeScript Tests ---"
+if [ -d "lambdas/pstn-transfer-ts" ]; then
+    if command -v npm &> /dev/null; then
+        cd lambdas/pstn-transfer-ts
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)